test/Sema/large_int_array.swift.gyb

```diff
@@ -0,0 +1,12 @@
+// RUN: %empty-directory(%t)
+// RUN: %gyb %s -o %t/large_int_array.swift
+// RUN: %target-swift-frontend -typecheck -verify %t/large_int_array.swift
```

Comment:
Could you add a version of this test that does a full compile? Just to make sure that any later stages of the compile pipeline can handle this.